s3bench.py

```diff
@@ -38,6 +38,9 @@ def __init__(self):
                             required=True)
         parser.add_argument('-w', '--workload', help='workload running on s3 - read/write',
                             required=True)
+        parser.add_argument('-l', '--max-latency',
+                            help='max acceptable latency per object operation in ms',
+                            required=False)
         parser.add_argument('-c', '--cleanup',
                             help='should we cleanup all the object that were written yes/no',
                             required=False)
```

Comment:
We should add here the flag for the prefix: 

```suggestion
parser.add_argument('-p', '--prefix', help='the given prefix under the bucket used for PUT/GET operation', required=False)
```